superset/viz.py

```diff
@@ -1186,6 +1186,8 @@ def process_data(self, df, aggregate=False):
                 df = df.rolling(**kwargs).sum()
         elif rolling_type == "cumsum":
             df = df.cumsum()
+        elif rolling_type == "cumperf":
+            df = (1.0 + df).cumprod() - 1.0
```

Comment:
Looking at the formula, I wonder if `cumret`, as in cumulative return, would be more appropriate?